### PR TITLE
Select value assertions

### PR DESCRIPTION
**Pull request by julioolvr**
I stumbled upon the issue mentioned in #80 and figured I could give it a try at fixing it. Very nice codebase 👏

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -851,7 +851,7 @@ $ npm run test:lint
 Tests:
 
 ```shell
-$ npm run test:tests
+$ npm run test:unit
```
Comment (ayrton) [inline README.md]: Oops 🙊

**Comment by ayrton** [comment]
👍

**Comment by ayrton** [comment]
Thanks for your contribution @julioolvr, let's wait for one other +1 until we merge in

**Comment by keithamus** [comment]
👍

**Pull request merged**